draft-xyz-rats-psa-endorsements.md

```diff
@@ -131,94 +141,173 @@ firmware measurement associated with Implementation ID
+  as illustrated in {{fig-cert-link}}.
+
+A PSA Certification data element has the following attributes:
+
+* `cert-level-type-choice`    : PSA certification level.  Acceptable values are x, y, z...
+* `cert-num-type`             : A unique number for each certificate
+* `cert-issue-date-type`      : Certificate date of issue, formatted as per RFC 3339 
+* `cert-test-lab-type`        : Name of the certication lab
+* `cert-holder-type`          : Name of Organization which holds the certificate
+* `cert-product-type`         : Name of the product been certified
+* `cert-hw-version-type`      : Version of Hardware certified
+* `cert-sw-version-type`      : Version of Software certified
+* `cert-type-type`            : Further details on certification
+* `cert-dev-type-type-choice` : Type of certificate, Chip, Device or System Software
+
+The `comid.psa-cert-meta` map is as follows:
 
-~~~ goat
-{::include examples/linked-tag.diag}
 ~~~
-{: #ex-linked-tag title="Example linked tag in a patch Reference Value CoMID"}
+{::include psa-ext/cert-meta.cddl}
+~~~
 
-#### Update Graph
+The provisioning linkage of Certification CoMID to the RoT CoMID is illustrated in {{fig-cert-link}}.
 
-{{fig-device-refval-updates}} illustrates the "update" graph for the firmware
-components associated with a given PSA hardware RoT.  Each node in the graph is
-the Reference Value CoMID associated with a specific version of one firmware
-component. Each tagged edge is the CoMID-to-CoMID link relation describing an
-update or patch action on the target node by the source node.
+~~~
+{::include art/cert.txt}
+~~~
+{: #fig-cert-link title="Certification Link to the RoT been certified"}
 
-~~~ goat
-{::include art/one-comid-per-refval-updates.txt}
+The example in {{ex-cert-ext}} shows a CoMID endorsement claim carrying a
+certification metadata associated to Implementation ID
+`acme-implementation-id-000000001`.
+
+~~~
+{::include examples/cert-val.diag}
 ~~~
-{: #fig-device-refval-updates title="Firmware update graph"}
+{: #ex-cert-ext title="Example Certification Claim"}
 
-### CoMID Layout
+The example in {{ex-cert-val-link}} shows the link entry to CoMID tag, been certified.
 
-Depending on the firmware update strategy, Endorsers have the following
-two packing options:
+~~~
+{::include examples/cert-val-link.diag}
+~~~
+{: #ex-cert-val-link title="Example Certification CoMID Linkage"}
 
-1. Put multiple Reference Values in one CoMID;
-2. Use one CoMID for each Reference Value.
+## PSA Provisioning Model
 
-The first packing option MAY be used if the device firmware is made of
-separately measurable components and these components are always installed
-together in a single software update operation.  If differential software
-updates are possible (or if there is only one measurable firmware component),
-Endorsers MUST use the second packing option.
+Two provisioning models are envisioned, for provisioning PSA Endorsements.
```

Comment:
```suggestion
Two provisioning models are envisioned for provisioning PSA Endorsements.
```

Reply:
I think , make sense here for a pause and sets context?

Reply:
it's one sentence, the comma is unneeded